app/src/Components/LandingPage/AdminModal/AdminModal.js

```diff
@@ -20,6 +20,7 @@ class AdminModal extends React.Component {
 
     updateToken = (fetchedToken) => {
         let updatedToken = fetchedToken;
+        // let updatedToken = 'jsklfjajsdlkfjla'; //use this for testing, uncomment above line in live, and comment out this one.
```

Comment:
Just a reminder to remove when your happy to merge :)

Reply:
Done.